scripts/plugin/plugin.yaml

```diff
@@ -11,13 +11,13 @@
 - name: "Markdown"
   version: "0.8.0"
 - name: "PieChart"
-  version: "0.8.0"
+  version: "0.8.1"
 - name: "Prometheus"
-  version: "0.51.0-rc.3"
+  version: "0.52.0"
```

Comment:
FYI: This new Prometheus version also will have this bug - https://github.com/perses/perses/issues/3084